vaadinfrontend/src/main/java/life/qbic/datamanager/views/projects/project/experiments/experiment/ExperimentDetailsComponent.java

```diff
@@ -60,55 +58,79 @@
 @UIScope
 @SpringComponent
 public class ExperimentDetailsComponent extends Composite<PageComponent> {
-
+  private ExperimentId experimentId;
+  private ExperimentInfoComponent experimentInfoComponent;
   private ExperimentalVariableCard experimentalVariableCard;
+  private ExperimentalVariablesComponent experimentalVariablesComponent;
   @Serial
   private static final long serialVersionUID = -8992991642015281245L;
-  private final transient Handler handler;
+  private final ExperimentInformationService experimentInformationService;
   private final HorizontalLayout tagLayout = new HorizontalLayout();
   private final TabSheet experimentSheet = new TabSheet();
-
   private final Div contentExperimentalGroupsTab = new Div();
-
-  private final Board summaryCardBoard = new Board();
-
   private final Div experimentSummary = new Div();
   private final ExperimentalGroupCardCollection experimentalGroupsCollection = new ExperimentalGroupCardCollection();
-  private final CardComponent sampleOriginCard = new CardComponent();
   private final CardComponent blockingVariableCard = new CardComponent();
   private final Button addBlockingVariableButton = new Button("Add");
   private final AddVariablesDialog addVariablesDialog;
   private final AddExperimentalGroupsDialog experimentalGroupsDialog;
-
   private final DisclaimerCard noExperimentalVariablesDefined;
-
   private final CreationCard experimentalGroupCreationCard = CreationCard.create(
-      "Add Experimental Group");
+      "Add more experimental groups");
```

Comment:
MAJOR NIT: Since we currently allow only adding one group at a time this could be kept in singular?
```suggestion
      "Add experimental group");
```